tabular/src/autogluon/tabular/models/abstract/abstract_model.py

```diff
@@ -579,15 +583,24 @@ def hyperparameter_tune(self, X_train, y_train, X_val, y_val, scheduler_options,
                 if isinstance(params_copy[hyperparam], Space):
                     logger.log(15, f"{hyperparam}:   {params_copy[hyperparam]}")
 
+        epochs = kwargs.get('epochs', 1)
```

Comment:
Not sure I like the name 'epochs'. This has a pretty specific meaning for NN models, and this here is an abstract model class.

Reply:
I can think of names like "rounds" or "train_iters". Can we decide one first before I update the code? This would require updates at multiple places. So I'd like the name to be fixed first. Epoch is actually quite close to the actual meaning, since we iterate this number of times to add trees to different base learners.